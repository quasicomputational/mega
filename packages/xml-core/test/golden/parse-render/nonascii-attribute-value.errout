Warning: Non-ASCII literal characters at (1:11-1:12).
