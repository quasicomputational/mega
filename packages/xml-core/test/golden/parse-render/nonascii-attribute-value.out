<foo bar="&#x3042;"></foo>